include/csoundCore.h

```diff
@@ -575,7 +781,7 @@ extern int ISSTRCOD(MYFLT);
     uint64_t kcounter;
     MYFLT    esr, sicvt, pidsr;                  /* local sr */
     MYFLT    onedsr;
-    int     overmode;
+    int     in_cvt, out_cvt;
```

Comment:
BTW: I'm trying to decipher what in_cvt and out_cvt means. It's not clear to me what it stands for.

Reply:
these are converter modes for in and out instead of a single overmode. Labelled for reference.